pull_request_template.md

```diff
@@ -1,13 +1,18 @@
 ### What this PR does (required):
-- list PR functionality here
+
```

Comment:
Please avoid editing the PR template, you can instead enter a description after you've successfully created a PR on Github :)